tests/incremental/hashes/function_interfaces.rs

```diff
@@ -279,7 +279,7 @@ pub fn no_mangle() {}
 #[rustc_clean(cfg = "cfail3")]
 #[rustc_clean(cfg = "cfail5")]
 #[rustc_clean(cfg = "cfail6")]
-#[no_mangle]
+#[unsafe(no_mangle)]
```

Comment:
The extra unsafe is necessary because without it these tests produces a lint, meaning fail2 and cfail5 are not clean. This was not previously a problem because the lint was emitted early, while it is now stored in the hir.